styleguide-app/AtomicDesignType.elm

```diff
@@ -0,0 +1,50 @@
+module AtomicDesignType exposing
+    ( AtomicDesignType(..)
+    , sorter
+    , toString
+    )
+
+{-|
+
+@docs AtomicDesignType
+@docs sorter
+@docs toString
+
+-}
+
+import Sort exposing (Sorter)
+
+
+{-| -}
+type AtomicDesignType
+    = Atom
+    | Molecule
+    | Organism
+    | Template
+    | Page
+
+
+{-| -}
+sorter : Sorter AtomicDesignType
+sorter =
+    Sort.by toString Sort.alphabetical
```

Comment:
💭 this will sort Atom, Molecule, Organism, **Page**, Template. Should it sort Page last?